<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(teste.jpg);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>250</y>
      <width>361</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QFrame{
background:#333;

border-bottom:5px solid white;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{

color:white;
border:none;
 font-size:22px;
 font-family:century gothic;
}</string>
     </property>
     <property name="text">
      <string>MENU AGENDAMENTO</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>330</y>
      <width>361</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QFrame{
background:#333;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_agendamento">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius:15px;
	background:rgb(70, 70, 70);
	font-size:20px;
	font-family:century gothic;
	color:white;
}
</string>
     </property>
     <property name="text">
      <string>AGENDAR VACINAÇÃO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_logout">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:#333;
border:3px;
font-size:13px;
font-family:century gothic;
color:white;
}

QPushButton:hover {
	border:1px solid rgb(93, 93, 93)

}</string>
     </property>
     <property name="text">
      <string>LOGOUT</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>351</width>
       <height>111</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>CPF</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Estado</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cidade</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Endereco</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Data</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>280</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;
background:green;
font-size:20px;
font-family:century gothic;
color:white;
</string>
     </property>
     <property name="text">
      <string>Atualizar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{

color:white;
border:none;
 font-size:22px;
 font-family:century gothic;
}</string>
     </property>
     <property name="text">
      <string>AGENDAMENTO CADASTRADO</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
